docs/developer/plugin-list.asciidoc

```diff
@@ -931,6 +931,12 @@ in their infrastructure.
 |The task manager is a generic system for running background tasks.
 
 
+|{kib-repo}blob/{branch}/x-pack/platform/plugins/private/task_manager_dependencies/README.md[taskManagerDependencies]
+|This plugin is used as a temporary sidecar plugin to enable the task manager plugin access to
```

Comment:
Do we have an issue open for this refactoring? I think everywhere we reference the temporariness of this plugin, we should link to the issue.